system/modules/core/templates/modules/mod_breadcrumb.html5

```diff
@@ -2,14 +2,14 @@
 
 <?php $this->block('content'); ?>
 
-  <ul>
+  <ol itemscope itemtype="http://schema.org/BreadcrumbList">
     <?php foreach ($this->items as $item): ?>
       <?php if ($item['isActive']): ?>
-        <li class="active<?php if ($item['class']): ?> <?php echo $item['class']; ?><?php endif; ?> last"><?php echo $item['title']; ?></li>
+        <li class="active<?php if ($item['class']): ?> <?php echo $item['class']; ?><?php endif; ?> last" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem"><span itemprop="item name"><?php echo $item['title']; ?></span></li>
```

Comment:
`itemprop="name"` is a child of `itemtype="http://schema.org/ListItem"` so this has to be wrapped inside an extra span